addons/behaviour_toolkit/behaviour_tree/leaves/leaf_signal.gd

```diff
@@ -0,0 +1,19 @@
+@icon("res://addons/behaviour_toolkit/icons/BTLeafSignal.svg")
+class_name LeafSignal
```

Comment:
I used this syntax in every script:
```gdscript
class_name LeafSignal extends BTLeaf
```
I would prefer this over a multline approach, as it makes the header smaller and more readable.